lib/launch_core/net/udp.c

```diff
@@ -130,7 +130,7 @@ int l_send_udp_broadcast(int sock, const uint8_t *buff, size_t len, uint16_t por
     if (ret == 0) {
         ret = sendto(sock, buff, len, 0, (struct sockaddr *) &addr, sizeof(addr));
         if (ret < 0) {
-            LOG_ERR("Failed to send broadcast message (%d)", ret);
+            LOG_ERR("Failed to send broadcast message (%d), (%d)", ret, errno);
```

Comment:
Do errno and ret not equal the same thing? As in you return an errno which gets assigned to ret anyways or is there a bug here

Reply:
I didn't mean to leave this here.

Reply:
No they are not the same.

Reply:
https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/19a63910114d5e157da63fb13b5e9193e52c49a2